integration_tests/src/main/python/spark_init_internal.py

```diff
@@ -121,6 +121,7 @@ def pytest_sessionstart(session):
     # can be reset in the middle of a test if specific operations are done (some types of cast etc)
     _sb = pyspark.sql.SparkSession.builder
     _sb.config('spark.plugins', 'com.nvidia.spark.SQLPlugin') \
+            .config("spark.sql.ansi.enabled", "false") \
```

Comment:
If we do this we have to have an issue to come back and add ANSI tests for everything that might have different behavior for ANSI vs not.

Reply:
https://github.com/NVIDIA/spark-rapids/issues/11047